tools/cockpit.spec

```diff
@@ -509,6 +509,13 @@ restorecon /usr/libexec/cockpit-wsinstance-factory
 %postun ws
 %systemd_postun_with_restart cockpit.socket
 %systemd_postun_with_restart cockpit.service
+if [ "$1" = "0" ]; then
+  # When $1 is 0, this is an uninstallation.
+  getent passwd cockpit-ws >/dev/null && userdel cockpit-ws
+  getent group cockpit-ws >/dev/null && groupdel cockpit-ws
+  getent passwd cockpit-wsinstance >/dev/null && userdel cockpit-wsinstance
+  getent group cockpit-wsinstance >/dev/null && groupdel cockpit-wsinstance
```

Comment:
This should be safe right now, provided that this makes sure that uninstalling the package stops `cockpit.service` first. If we land this, it needs to be accompanied by an integration test (in check-login or so) that validates this. This should then also be done in the Debian packaging in tools/debian/.

However, I wonder if it's worth the potential trap to clean up half of the system user.